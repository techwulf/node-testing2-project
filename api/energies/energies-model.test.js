
const db = require('../../data/dbConfig');
const Energies = require('./energies-model');

test('environment', () => {
  expect(process.env.NODE_ENV).toBe('testing');
});

beforeAll(async () => {
  await db.migrate.rollback();
  await db.migrate.latest();
});

beforeEach(async () => {
  await db.seed.run();
});

afterAll(async () => {
  await db.destroy();
});

describe('Energies.getAll()', () => {
  let energies;
  beforeEach(async () => {
    energies = await Energies.getAll();
  })

  it.todo('returns all (4) energies');
  it.todo('energies returned are in the correct shape');
});