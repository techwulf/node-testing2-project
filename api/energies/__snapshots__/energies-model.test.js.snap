// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Energies.getAll() energies returned are in the correct shape 1`] = `
Array [
  Object {
    "energy": "demonic",
    "id": 1,
  },
  Object {
    "energy": "fey",
    "id": 2,
  },
  Object {
    "energy": "celestial",
    "id": 3,
  },
  Object {
    "energy": "undead",
    "id": 4,
  },
]
`;
